### DISCUSSION TITLE
[@types/mime] No longer needed starting with `mime@4`

### DISCUSSION THREAD

**Discussion by broofa** [DefinitelyTyped/DefinitelyTyped#69200] [Issues with a @types package]
I maintain the `mime` module.   Typedefs are now bundled with it, starting with `mime@4`.

Are there instructions for how to deprecate `@types/mime`?  I could put together a PR but would be happy to let someone with prior knowledge do that. :-)

Announcement: https://github.com/broofa/mime?tab=readme-ov-file#mime (see "Notes")

Related issue: https://github.com/broofa/mime/issues/264

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
See this section of the README for information: <https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#removing-a-package>

**Comment by ghost** [comment]
Removed in #69213
